.github/workflows/bump2release.yml

```diff
@@ -0,0 +1,82 @@
+name: Bump version to release
+
+on:
+  workflow_dispatch:
+    inputs:
+      MAJOR_OR_MINOR:
+        type: choice
+        description: major/minor release
+        options:
+          - minor
+          - major
+        default: minor
+        required: true
+      VERSION_NAME:
+        type: string
+        description: major version name
+        default: ''
+
+jobs:
+  bump_version:
+    runs-on: ubuntu-latest
+    env:
+      GIT_USERNAME: "No-Reply"
+      GIT_USEREMAIL: "no-reply@endlessos.org"
```

Comment:
I think you can use `${{ github.actor }}` and you'll get the github username of the person triggering the workflow. Then I think you can use that as the username and probably it could be used in the made up email address, too.

See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context for details.

Reply:
good idea!